tools/star.wdl

```diff
@@ -28,22 +28,28 @@ task build_star_db {
         sjdb_gtf_tag_exon_parent_gene_type: "GTF attribute name for parent gene type"
         use_all_cores: {
             description: "Use all cores? Recommended for cloud environments.",
-            group: "common",
+            group: "Resources",
```

Comment:
I think that `Resources` is fine for STAR, since it has an overwhelming number of options. I'm not sure if it is something that we should pursue across our repo though. It would be a little annoying to be inconsistent, but I'm not sure for smaller argument sets that splitting `Resources` out from `Common` is useful.

Reply:
Hmm. I don't love being inconsistent, but I'm also not looking forward to this mass update and am happy to skip it 😅 

FWIW, in `wdl-doc`, I've written it so that `Resources` group is hardcoded to appear as _the last "group" of inputs_ (and `Common` is hardcoded to come first)

Reply:
@adthrasher I suppose I'm ok with leaving it inconsistent for now, and we can always come back at a later date. I'm going to mark this ready for review so we can get it merged